app/models/course/material/folder/download.rb

```diff
@@ -0,0 +1,7 @@
+class Course::Material::Folder::Download < ActiveRecord::Base
```

Comment:
No, that's just because ActiveJob generates UUIDs...

Wait a while for this, let me check.